DependencyInjection/Compiler/AddDependencyCallsCompilerPass.php

```diff
@@ -140,6 +140,30 @@ public function process(ContainerBuilder $container)
                     $groups[$resolvedGroupName]['roles'] = $groupDefaults[$resolvedGroupName]['roles'];
                 }
             }
+        } elseif ($container->getParameter('sonata.admin.configuration.sort_admins')) {
+            $groups = $groupDefaults;
+
+            $label_or_admin = function ($e) {
+                return !empty($e['label']) ? $e['label'] : $e['admin'];
+            };
+
+            // sort the default group definition
+            ksort($groups);
+            array_walk(
```

Comment:
Reup of https://github.com/sonata-project/SonataAdminBundle/pull/3041#discussion_r32021097.

> Just asking: What is the goal of this function usage? Could not be more simple?

ping @rande